natives.json

```diff
@@ -115491,16 +115491,16 @@
 			"build": "1207"
 		},
 		"0x0C3F0F7F92CA847C": {
-			"name": "_0x0C3F0F7F92CA847C",
+			"name": "_SET_VEHICLE_SPEED",
```

Comment:
This is for draft vehicles (``VEHICLE_TYPE_DRAFT``) only.
So this should be changed to something like ``_SET_DRAFT_VEHICLE_SPEED`` which also fits alphabetically.

Useful information would be if this is similar to ``SET_VEHICLE_FORWARD_SPEED`` or if it sets some sort of "desired" speed.

Reply:
It's not similar to `SET_VEHICLE_FORWARD_SPEED`, it's a sort of "desired" speed.

Reply:
I guess it should be named ``_SET_DRAFT_VEHICLE_DESIRED_SPEED`` then.